CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # SAS Viya Monitoring for Kubernetes
 
+## UNRELEASED
+
+* **Overall**
+
+* **Metrics**
+
+* **Logging**
+  * [FIX] Updated Fluent Bit configuration to use `Allowlist_key` instead of `Whitelist_key` modifier as it has been deprecated and is ignored.
```

Comment:
I think we should make a process change to start including CHANGELOG changes with our PRs. This will make end of sprint activities a bit easier and just have us update the version at the top

Reply:
I think that is a good idea.  Let's try it out in this sprint and see if merge conflicts (due to CHANGELOG changes) become an issue.

Reply:
Sounds good!